tutorials/tabs-manager/popup.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./popup.css" />
+  </head>
+  <body>
+    <template id="li_template">
+      <li>
+        <a>
+          <h3 class="title">Tab Title</h3>
+          <p class="pathname">Tab Pathname</p>
+        </a>
+      </li>
+    </template>
+
+    <h1>Google Dev Docs</h1>
+    <button>Group Tabs</button>
+    <ul></ul>
+
+    <script src="./popup.js" type="module"></script>
```

Comment:
@AmySteam Although it is not specific to Chrome extensions, I discovered we can use [top level await] in the background page. It could be worth to highlight it in the tutorial. What do you think?

[Top level await]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules#top_level_await

Reply:
@alexherbo2 That's a great idea! I'll include it in the tutorial 👍 

I really appreciate it that you are taking time to share your ideas. Please feel free to comment on any of the tutorials :)